targets/LuaSdk/SharedTesting/PlayFabTesting/PlayFabApiTestSuite.lua

```diff
@@ -122,6 +122,7 @@ end
 function PlayFabApiTestSuite.OnLoginSuccess(result)
     if (result.PlayFabId) then 
         PlayFabApiTestSuite.playFabId = result.PlayFabId
+        PlayFabSettings._internalSettings.sessionTicket = result.SessionTicket
```

Comment:
Same deal here as with entityToken.
Please remove this sessionTicket assignment here, and apply this assignment somewhere appropriate in the PlayFabApi.lua.ejs file. Doing it here means it's only set once for this test, and not properly set "in general" which really defeats the entire point of this test.